src/fs/ufs/UFSStoreState.cc

```diff
@@ -95,7 +95,7 @@ Fs::Ufs::UFSStoreState::closeCompleted()
            theFile->error());
 
     if (theFile->error()) {
-        debugs(79,3,HERE<< "theFile->error() ret " << theFile->error());
+        debugs(79,3,"theFile->error() ret " << theFile->error());
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.